packages/lab/src/color-chooser/ColorChooser.tsx

```diff
@@ -144,7 +146,7 @@ export const ColorChooser = ({
     setActiveTab(index);
   };
 
-  const OverlayData = () => {
```

Comment:
Did this renaming previously, but think OverlayContent makes more sense than OverlayData, same as seen in the contact details pattern